front/src/Components/AuthentificationForm.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import ParameterCardTitle from './ParameterCard/ParameterCardTitle';
+import Grid from '@mui/material/Grid/Grid';
+import { Link } from "react-router-dom";
+import { useFormManager } from "react-simple-form-manager";
+
+interface AuthentificationFormState {
+  username: string;
+  password: string;
+}
+
+const minimumFieldLength = 8;
+
+const fieldValidator = (field: string) => {
```

Comment:
Maybe separate fieldValidator of username and password, might want a username shorter than 8 characters like "Bluub"